azure-mgmt-batch/src/main/java/com/microsoft/azure/management/batch/Application.java

```diff
@@ -0,0 +1,190 @@
+         */
+        interface Blank<ParentT> extends WithAttach<ParentT> {
+        }
+
+        /** The final stage of the application definition.
+         * <p>
+         * At this stage, any remaining optional settings can be specified, or the application definition
+         * can be attached to the parent batch account definition using {@link Application.DefinitionStages.WithAttach#attach()}.
+         * @param <ParentT> the return type of {@link Application.DefinitionStages.WithAttach#attach()}
+         */
+        interface WithAttach<ParentT> extends
+                Attachable.InDefinition<ParentT> {
+
+            /**
+             * Set the allowUpdates value.
+             *
+             * @param allowUpdates the allowUpdates value to set
+             * @return parent batch account definition.
+             */
+            DefinitionStages.WithAttach<ParentT> withAllowUpdates(boolean allowUpdates);
+
+            /**
+             * Set the displayName value.
+             *
+             * @param displayName the displayName value to set
+             * @return parent batch account definition.
+             */
+            DefinitionStages.WithAttach<ParentT> withDisplayName(String displayName);
+        }
+    }
+
+    /**
+     * The entirety of a virtual machine extension definition as a part of parent update.
+     * @param <ParentT> the return type of the final {@link Attachable#attach()}
+     */
+    interface UpdateDefinition<ParentT> extends
+            UpdateDefinitionStages.Blank<ParentT>,
+            UpdateDefinitionStages.WithAttach<ParentT> {
+    }
+
+    /**
+     * Grouping of application definition stages as part of parent batch account update.
+     */
+    interface UpdateDefinitionStages {
+        /**
+         * The first stage of a virtual machine extension definition.
+         *
+         * @param <ParentT> the return type of the final {@link WithAttach#attach()}
+         */
+        interface Blank<ParentT>
+                extends WithAttach<ParentT> {
+        }
+
+        /** The final stage of the application definition.
+         * <p>
+         * At this stage, any remaining optional settings can be specified, or the application definition
+         * can be attached to the parent batch account definition using {@link Application.DefinitionStages.WithAttach#attach()}.
+         * @param <ParentT> the return type of {@link Application.DefinitionStages.WithAttach#attach()}
+         */
+        interface WithAttach<ParentT> extends
+            Attachable.InUpdate<ParentT> {
+
+            /**
+             * Set the allowUpdates value.
+             *
+             * @param allowUpdates the allowUpdates value to set
+             * @return parent batch account update definition.
+             */
+            UpdateDefinitionStages.WithAttach<ParentT> withAllowUpdates(boolean allowUpdates);
+
+            /**
+             * Specify the display name.
```

Comment:
in general, for method description, we use the declarative mode not imperative, so it should say "Specifies" not "Specify"